### PR TITLE
Mention importing process more explicitly

### PR DESCRIPTION
**Pull request by JesseWeinstein**

### CODE REVIEW

**Comment by JesseWeinstein** [comment]
Fixed the target.

File: editions/tw5.com/tiddlers/wikitext/Images in WikiText.tid [hunk 1]
```diff
@@ -51,3 +51,7 @@ Renders as:
 
 {{Motovun Jack.jpg}}
 
+! Importing Images
+
+Use the {{$:/core/images/import-button}} ''import'' button (under the ''Tools'' tab in the sidebar), or drag and drop.
```
Comment (00SS) [inline editions/tw5.com/tiddlers/wikitext/Images in WikiText.tid]: Please consider using: `<<.button import>>`  & `<<.sidebar-tab Tools>>`

File: editions/tw5.com/tiddlers/wikitext/Images in WikiText.tid [hunk 2]
```diff
@@ -51,3 +51,7 @@ Renders as:
...
+See ImportingTiddlers for details.
```
Comment (00SS) [inline editions/tw5.com/tiddlers/wikitext/Images in WikiText.tid]: [[Importing Tiddlers]]

**Comment by JesseWeinstein** [comment]
These suggestions make sense, I just haven't had time to get to them.

**Comment by JesseWeinstein** [comment]
And now, many months later, I've finally found the time! Hopefully this can get merged now? :-)

**Comment by JesseWeinstein** [comment]
8 months later, I came across this again -- presumably it fell thru the cracks for you, too? :-)

**Comment by Jermolene** [comment]
Thanks @JesseWeinstein, and many apologies, it had indeed fallen quite far between the cracks...

**Pull request merged**